source/Calamari.Tests/KubernetesFixtures/Authentication/SetupKubectlAuthenticationFixture.cs

```diff
@@ -96,7 +97,7 @@ public void Setup()
             });
             kubectl = Substitute.For<IKubectl>();
             kubectl.ExecutableLocation.Returns("kubectl");
-            kubectl.TrySetKubectl().Returns(true);
```

Comment:
Removed this `TryX` semantics and simplified to `X` and exceptions are thrown if things aren't configured as we would expect